packages/neuron-ui/src/components/Settings/InputWalletPswDialog.tsx

```diff
@@ -22,16 +23,24 @@ const InputWalletPswDialog = ({
   const [errorMsg, setErrorMsg] = useState('')
   const [password, setPassword] = useState('')
   const [t] = useTranslation()
+
   const handleSubmit = () => {
-    if (password) {
-      dispatch({
-        type: MainActions.SetDialog,
-        payload: null,
-      })
-      handle()
-    } else {
+    if (!password) {
       setErrorMsg('Please enter password')
     }
+    checkPassword(password, (passwordValid: boolean) => {
```

Comment:
Is it designed that only one password for all wallets? If not, the wallet identifier should be passed too.

Reply:
ok,make sense

Reply:
We didn't mention this yet, but for now let's assume we have separate password for each wallets.

In fact, all operations on a wallet should pass the wallet identifier. Although we have a `CurrentWallet` concept, that should only used to get current active wallet's identifier (or object for simple read) and the identifier should pass to low level.